diff -up simplelogin-master/CMakeLists.txt.1~ simplelogin-master/CMakeLists.txt
--- simplelogin-master/CMakeLists.txt.1~	2021-04-01 00:50:05.799160514 +0200
+++ simplelogin-master/CMakeLists.txt	2021-04-01 00:53:09.931831054 +0200
@@ -44,5 +44,5 @@ install(FILES upstart/simplelogin.conf D
 install(FILES upstart/simplelogin.override DESTINATION /etc/init)
 add_subdirectory(systemd)
 
-install(FILES sddm-autologin DESTINATION /etc/pam.d)
+install(FILES simplelogin DESTINATION /etc/pam.d)
 
diff -up simplelogin-master/HelperApp.cpp.1~ simplelogin-master/HelperApp.cpp
--- simplelogin-master/HelperApp.cpp.1~	2020-12-04 03:09:28.000000000 +0100
+++ simplelogin-master/HelperApp.cpp	2021-04-01 00:50:05.799160514 +0200
@@ -75,7 +75,7 @@ namespace SDDM {
         
         PamHandle *pamHandle = new PamHandle; //TODO fix leak
 
-        if (! pamHandle->start("sddm-autologin" /*PAM session*/, m_user)) //Martin check this exists
+        if (! pamHandle->start("simplelogin" /*PAM session*/, m_user)) //Martin check this exists
             qFatal("Could not start PAM");
 
         QProcessEnvironment env = m_session->processEnvironment();
diff -up simplelogin-master/README.1~ simplelogin-master/README
--- simplelogin-master/README.1~	2020-12-04 03:09:28.000000000 +0100
+++ simplelogin-master/README	2021-04-01 00:50:05.799160514 +0200
@@ -12,9 +12,9 @@ or sssdddmmmm for short.
 
 
 Note
-this file expects sddm-autologin in /etc/pam.d/
+this file expects simplelogin in /etc/pam.d/
 Use the ubuntu one, not the one from the repo. Ubuntu have a messed up system
 
 renaming the lightdm one should be fine too
 
-exec line can only be one string due to shoddy parsing, if you need to pass args, just go via a shell script or modify the code
\ No newline at end of file
+exec line can only be one string due to shoddy parsing, if you need to pass args, just go via a shell script or modify the code
diff -up simplelogin-master/simplelogin.1~ simplelogin-master/simplelogin
--- simplelogin-master/simplelogin.1~	2021-04-01 00:50:05.800160513 +0200
+++ simplelogin-master/simplelogin	2021-04-01 00:50:05.800160513 +0200
@@ -0,0 +1,16 @@
+#%PAM-1.0
+auth        required    pam_env.so
+auth        required    pam_shells.so
+auth        required    pam_nologin.so
+auth        required    pam_permit.so
+-auth       optional    pam_gnome_keyring.so
+-auth       optional    pam_kwallet5.so
+account     include     system-auth
+password    include     system-auth
+session     required    pam_loginuid.so
+session     optional    pam_keyinit.so force revoke
+session     required    pam_namespace.so
+session     include     system-auth
+-session    optional    pam_gnome_keyring.so auto_start
+-session    optional    pam_kwallet5.so auto_start
+session     include     postlogin
