diff -up simplelogin-master/systemd/simplelogin.service.2~ simplelogin-master/systemd/simplelogin.service
--- simplelogin-master/systemd/simplelogin.service.2~	2021-04-01 00:54:25.473695923 +0200
+++ simplelogin-master/systemd/simplelogin.service	2021-04-01 00:54:46.291658683 +0200
@@ -7,7 +7,7 @@ StartLimitIntervalSec=30
 StartLimitBurst=2
 
 [Service]
-ExecStart=/usr/bin/simplelogin --user phablet --session /usr/share/wayland-sessions/plasma-mobile.desktop
+ExecStart=/usr/bin/simplelogin --user omv --session /usr/share/wayland-sessions/plasma-mobile.desktop
 Restart=always
 
 [Install]
